broker/consumer/vm_startup.sh

```diff
@@ -87,17 +87,28 @@ do
     # which kills the while loop. no working suggestions found.
     # passing the error with `|| true`
 
+    
+    IFS=', ' read -r -a KAFKA_TOPIC_ARRAY <<< "$KAFKA_TOPIC"
+    KAFKA_TOPIC_PRESENT=()
+
     # check if our topic is in the list
-    if grep -Fq "${KAFKA_TOPIC}" "${fout_topics}"
-    then
-        alerts_flowing="true"  # start consuming
+    for topic in "${KAFKA_TOPIC_ARRAY[@]}"; 
+    do
+        if grep -Fq "${topic}" "${fout_topics}"
+        then
+            KAFKA_TOPIC_PRESENT+=("True")
+        fi
+    done
+    
+    if [ ${#KAFKA_TOPIC_PRESENT[@]} -eq ${#KAFKA_TOPIC_ARRAY[@]} ]; then
+        alerts_flowing="true"
     else
-        sleep 60s  # sleep 1 min, then try again
+        sleep 60s
     fi
 done
 
 #--- Start the Kafka -> Pub/Sub connector, save stdout and stderr to file
 /bin/connect-standalone \
     "${consumerdir}/psconnect-worker.properties" \
     "${consumerdir}/ps-connector.properties" \
-    &>> "${fout_run}"
+    &>> "${fout_run}"
```

Comment:
This is not a real change.

Reply:
I've noticed that this has been happening more frequently in a few different PRs. I'll make sure to review my changes before committing.

Reply:
Do you have any suggestions on how to fix this?